homeassistant/components/bthome/sensor.py

```diff
@@ -47,6 +47,15 @@
 from .device import device_key_to_bluetooth_entity_key
 
 SENSOR_DESCRIPTIONS = {
+    # Acceleration (m/s²)
+    (
+        BTHomeSensorDeviceClass.ACCELERATION,
+        Units.ACCELERATION_METERS_PER_SQUARE_SECOND,
+    ): SensorEntityDescription(
+        key=f"{BTHomeSensorDeviceClass.ACCELERATION}_{Units.ACCELERATION_METERS_PER_SQUARE_SECOND}",
+        native_unit_of_measurement=Units.ACCELERATION_METERS_PER_SQUARE_SECOND,
```

Comment:
This should be a Home Assistant unit

Reply:
But what should I use if there isn’t an Home Assistant unit of measurement m/s2.

Reply:
I didn't realize there wasn't a built-in one for this. Its ok to use a custom one in this case 👍